makelib/variables.mk

```diff
@@ -14,7 +14,7 @@ FILELISTDIR := $(BUILDDIR)/filelists
 MAINTEMPDIR := $(BUILDDIR)/tmp
 CLEANDIR := $(BUILDDIR)/clean
 
-ACTOOL := $(BINDIR)/actool
+ACTOOL := $(TOOLSDIR)/actool
```

Comment:
We have to make sure that release guide is updated about the new location of actool. I don't remember if we run some script to assemble the final tarball with rkt binary and whatnot, but if so, it also needs to be updated.

Other question is whether actool should be distributed at all.